web/src/app/browser/src/hardwareEventKeyboard.ts

```diff
@@ -38,15 +38,13 @@ export function _GetEventKeyCode(e: KeyboardEvent) {
 // Keeping this as a separate function affords us the opportunity to unit-test the method more simply.
 
 /**
- * Function     _GetKeyEventProperties
- * Scope        Private
- * @param       {Event}       e         Event object
- * @return      {Object.<string,*>}     KMW keyboard event object:
- * Description  Get object with target element, key code, shift state, virtual key state
- *                Lcode=keyCode
- *                Lmodifiers=shiftState
- *                LisVirtualKeyCode e.g. ctrl/alt key
- *                LisVirtualKey     e.g. Virtual key or non-keypress event
+ * Function     preprocessKeyboardEvent
+ * @param       {Event}         e              Event object
+ * @param       {KeyboardState} keyboardState  Keyboard state object
+ * @param       {DeviceSpec}    device         Device object
+ * @return      {KeyEvent}                     KMW keyboard event object:
+ *
+ * Description  Returns the preprocessed KeyEvent object.
```

Comment:
This doesn't really say anything. Can we explain what preprocessing does instead? (Also drop "Function" and "Description" as they don't match our [doc style](https://github.com/keymanapp/keyman/wiki/Keyman-Code-Style-Guide#Source-Code-Documentation-Syntax)